script/_otbr

```diff
@@ -58,7 +59,7 @@ otbr_install()
 
     (mkdir -p "${OTBR_TOP_BUILDDIR}" \
         && cd "${OTBR_TOP_BUILDDIR}" \
-        && cmake -GNinja "${OTBR_TOP_SRCDIR}" "${otbr_options[@]}" \
+        && cmake -GNinja "${OTBR_TOP_SRCDIR}" "${otbr_options[@]}" ${OTBR_OPTIONS[@]} \
```

Comment:
Double quote array expansions to avoid re-splitting elements.
```suggestion
        && cmake -GNinja "${OTBR_TOP_SRCDIR}" "${otbr_options[@]}" "${OTBR_OPTIONS[@]}" \
```